src/encoded/types/biosample.py

```diff
@@ -42,6 +42,28 @@ class Biosample(Item, CalculatedDonors):
         'treatments'
     ]
 
+    @calculated_property(schema={
+        "title": "Summary",
+        "description": "A summary of the treatments applied to the Biosample.",
+        "comment": "Do not submit. This is a calculated property",
+        "type": "string",
+        "notSubmittable": True,
+    })
+    def summary(self, request, biosample_ontology, derived_from):
+        summ = ''
+        my_type = self.item_type.replace('_',' ')
+        if my_type == 'organoid':
+            dfrom = set()
+            for df in derived_from:
+                df_obj = request.embed(df, '@@object?skip_calculated=true')
+                df_bo = df_obj.get('biosample_ontology')
+                df_bo_obj = request.embed(df_bo, '@@object?skip_calculated=true')
```

Comment:
the code is correct.
abbreviations can make the code hard to read though, here's one suggestion:

obj = request.embed(df, '@@object?skip_calculated=true')
ontology = obj.get('biosample_ontology')
ontology_obj = request.embed(ontology, '@@object?skip_calculated=true')

Reply:
updated